.changeset/stale-kids-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@evidence-dev/evidence': patch
+---
+
+Removed EVIDENCE_DISABLE_INCLUDE, behavior is now forced by `--debug`
```

Comment:
Would be better to leave these separate in my opinion. There are situations where we want `EVIDENCE_DISABLE_INCLUDE` without `--debug`, and vice versa.

Reply:
@ItsMeBrianD What do you think about this?

Reply:
Reverted this change: [0853ee0](https://github.com/evidence-dev/evidence/pull/2738/commits/0853ee02dbac3f3deac0d69a974be9afca59a953)

We can discuss more later if necessary